<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFTPApp</class>
 <widget class="QMainWindow" name="TFTPApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TFTPApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cliente</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Host:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leIPHost">
             <property name="text">
              <string>192.168.0.12</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Puerto:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spPort">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>69</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Archivo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leFileName">
           <property name="text">
            <string>Fly.jpg</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pbGet">
           <property name="text">
            <string>Get</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPut">
           <property name="text">
            <string>Put</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbStopClient">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Stop Client</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTFTP">
    <property name="title">
     <string>TFTP</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuTFTP"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
